[MAINTENANCE] Update stale.yml

AFineDayFor opened the pull request:
[contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:

Enable the little one :)

Previous Design Review notes:

StaleBot should be used to prevent the DevRel log from artificially inflating.
Definition of Done

Please delete options that are not relevant.

My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style) -- actually I include the leading ---for yaml files ^_^.
I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
I have commented my code, particularly in hard-to-understand areas
I have run any local integration tests and made sure that nothing is broken. yamllint everything.
(🌸◕‿◕)
Cheers

AFineDayFor committed:
[MAINTENANCE] Update stale.yml

scope and time-frame.

ghost commented:
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=5602&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=5602&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)
</details>

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/great-expectations/great_expectations/5602/7f65b2ea/b12790904bd234ddfd0a9a957055490e1be96e0b.svg)](https://app.codesee.io/r/reviews?pr=5602&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

AFineDayFor commented on .github/workflows/stale.yml:
```diff
@@ -9,32 +9,34 @@ on:
         # Scope
         only-pr-labels: community
+        only-issue-labels: community
+        exempt-issue-labels: core-team 
```
"help wanted"

AFineDayFor replied:
done.

AFineDayFor committed:
add "help wanted" to exempt issue labels.

to prevent  https://github.com/great-expectations/great_expectations/issues/5265 falling into that list of stale.

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

AFineDayFor committed:
Added "DevRel Triage" to exempt issue labels. 

experience is a good teacher.

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

github-actions[bot] committed:
Merge develop into MAINTENANCE]-Stale/Scoping

AFineDayFor committed:
Merge branch 'develop' into MAINTENANCE]-Stale/Scoping
